### PR TITLE
feat: add dapp rewards shortcut card

### PR DESCRIPTION
**Pull request by kathaypacific**
### Description

This PR:
- add feature gate for dapp rewards shortcuts
- create a component that houses the carousel of cards (I've called them DappFeaturedActions, for lack of a better name.....suggestions welcome) and move dapp rankings card into there
- rename some components after moving things around so that they still make sense
- add TODO's for stuff that's not included in this PR

### Test plan

https://github.com/valora-inc/wallet/assets/20150449/6713a16b-13cc-4146-bad9-59ffc10bc6ac

### Related issues

- Fixes RET-733

### Backwards compatibility

y

### CODE REVIEW

File: src/statsig/constants.ts [hunk 1]
```diff
@@ -25,6 +25,7 @@ export const LayerParams = {
 export const FeatureGates = {
   [StatsigFeatureGates.USE_ZENDESK_API_FOR_SUPPORT]: false,
   [StatsigFeatureGates.SHOW_POSITIONS]: false,
+  [StatsigFeatureGates.SHOW_DAPP_SHORTCUTS]: false,
```
Comment (jeanregisser) [inline src/statsig/constants.ts:28]: nit: more specific so we can add new gates for more advances shortcuts in the future
```suggestion
  [StatsigFeatureGates.SHOW_CLAIM_SHORTCUTS]: false,
```

**Review by jeanregisser — APPROVED** [review]
Awesome! 👍

File: src/statsig/constants.ts (hunk 1 above)
Comment (kathaypacific) [reply src/statsig/constants.ts]: updated

File: src/dappsExplorer/DAppsExplorerScreenSearchFilter.tsx [hunk 2]
```diff
@@ -171,7 +172,7 @@ export function DAppsExplorerScreenSearchFilter() {
                   title={t('dappsScreen.title')}
                   message={t('dappsScreen.message')}
                 />
-                <DappRankingsCard onPress={handleShowDappRankings} />
+                <DappFeaturedActions onPressShowDappRankings={handleShowDappRankings} />
```
[the remark this answers is not in the document]
Comment (kathaypacific) [reply src/dappsExplorer/DAppsExplorerScreenSearchFilter.tsx:175]: yeah makes sense, although i didn't think we'd have that many actions in this carousel. i can make this change in a separate PR

**Pull request merged**